README.md

```diff
@@ -157,12 +157,6 @@ $ grunt bump:prerelease
 >> Tagged as "v1.0.0-1"
 >> Pushed to origin
 
-$ grunt bump:patch
```

Comment:
it was meant to show the prerelease version being bumped to a patch version, in order to demonstrate the git version bump. It might be more clear to add comments to what is happening if its not clear already